javascript/ql/src/semmle/javascript/dataflow/Sources.qll

```diff
@@ -9,15 +9,17 @@
 import javascript
 
 /**
- * A source node for local data flow, that is, a node for which local
- * data flow cannot provide any information about its inputs.
+ * A source node for local data flow, that is, a node from which local data flow is tracked.
  *
- * By default, functions, object and array expressions and JSX nodes
- * are considered sources, as well as expressions that have non-local
- * flow (such as calls and property accesses). Additional sources
- * can be modelled by extending this class with additional subclasses.
+ * Examples include function parameters, imports and property accesses; see
+ * `DataFlow::SourceNode::DefaultRange` for details. You can introduce new kinds of
```

Comment:
Minor:
Consistent docstring style: the use of "you" is very rare in ql/qll files. Should the sentence be something like the following instead?

> Define new subclasses of `DataFlow::SourceNode::Range` to introduce new kinds of source nodes.

Reply:
I modelled it loosely after the docstring for `InvokeNodeDef`, which also has "you".